test/unit/com/unifina/signalpath/charts/TimeSeriesChartSpec.groovy

```diff
@@ -62,6 +62,9 @@ class TimeSeriesChartSpec extends UiChannelMockingSpecification {
 		new ModuleTestHelper.Builder(module, inputValues, outputValues)
 			.timeToFurtherPerIteration(1000)
 			.uiChannelMessages(channelMessages, getSentMessagesByStreamId())
+			.afterEachTestCase {
+				module.headersSent = false
```

Comment:
`Chart` has set `canClearState = false`. Thus this manual invocation after each test case is necessary for the header message to be sent.